### PR TITLE
Documentation regarding absolute_fee and fee_rate updated

### PR DESCRIPTION
**Pull request by roy9495**
### Description
This pr solves this issue #856 
Updated the documentation for absolute fee and fee rate in BDK docs

#### All Submissions:

* [x] I've signed all my commits
* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)

#### Bugfixes:

* [x] I'm linking the issue being fixed by this PR

### CODE REVIEW

File: crates/bdk/src/wallet/tx_builder.rs [hunk 1]
```diff
@@ -190,6 +190,9 @@ impl<'a, D, Cs: CoinSelectionAlgorithm, Ctx: TxBuilderContext> TxBuilder<'a, D,
     }
 
     /// Set an absolute fee
+    /// The fee_absolute method refers to the absolute transaction fee in satoshis (sats).
+    /// If anyone set both the fee_absolute method and the fee_rate method,
```
Comment (notmandatory) [inline crates/bdk/src/wallet/tx_builder.rs]: grammar nit:
```suggestion
    /// If anyone sets both the fee_absolute method and the fee_rate method, 
```

**Comment by jonmarrs** [comment]
While we are updating the absolute fee documentation, perhaps we should also update the fee rate documentation for the fee_rate method directly above the fee_absolute method.

Is the fee rate in satoshi/vbyte (e.g., 5.0 satoshi/vbyte)?

**Comment by jonmarrs** [comment]
I've updated the documentation for the fee_rate method in this pull request: https://github.com/bitcoindevkit/bdk/pull/969

**Comment by danielabrozzoni** [comment]
Hi, can you please squash your commits together? Then I will merge this :)

**Comment by jonmarrs** [comment]
I've squashed all of the commits together here: https://github.com/bitcoindevkit/bdk/pull/969

**Comment by roy9495** [comment]
Hello, I have squashed my commits. Thank you.

**Review by notmandatory — APPROVED** [review]
ACK 17b3d65d4c05dfc1575627a60834acb6b4a921aa

**Comment by danielabrozzoni** [comment]
Hey, can you please rebase on master to pick up the CI fixes?

**Comment by roy9495** [comment]
I rebased the branch. Thank you

**Comment by danielabrozzoni** [comment]
I need your commits to be signed for merging: https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by roy9495** [comment]
I signed the commits. Thank you

**Review by danielabrozzoni — APPROVED** [review]
ACK 10b4b6c665d58321436413d1d9c0486e7d74c5ec

**Comment by roy9495** [comment]
Thank You

**Pull request merged**